allenact/setup.py

```diff
@@ -109,9 +109,9 @@ def get_version(fname):
         packages=find_packages(include=["allenact", "allenact.*"]),
         install_requires=[
             "gym>=0.17.0,<0.18.0",
-            "torch>=1.6.0,!=1.8.0",
+            "torch>=1.6.0,!=1.8.0,<1.9.0",
```

Comment:
Does `>=1.9.0` not work for us? I believe I've ran a few things using `1.9` with `allenact` without problems.

Reply:
1.9.0 was allocating extra memory on GPU 0 when using GRU/LSTM on other GPUs (Ubuntu 18.xx, CUDA driver 11-ish)

Reply:
Ah I did see this but thought it was my fault. Sounds good, let's keep an eye on this so we can try to remove this restriction as soon as they have a fix.